packages/datatrak-web/index.html

```diff
@@ -8,6 +8,30 @@
       property="og:description"
       content="Data aggregation, analysis, and visualisation for the most remote settings in the world"
     />
+    <% if (process.env.REACT_APP_DEPLOYMENT_NAME === 'master' ||
+    process.env.REACT_APP_DEPLOYMENT_NAME === 'main' || process.env.REACT_APP_DEPLOYMENT_NAME ===
+    'production') { %>
+      <!-- Google Analytics -->  
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-P5KF6SLGKR"></script>
+        <script>
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+
+          gtag('config', 'G-P5KF6SLGKR');
+        </script>
+      <!-- End Google Analytics -->
+
+      <!-- Clarity Analytics -->
+        <script type="text/javascript">
+          (function(c,l,a,r,i,t,y){
+              c[a]=c[a]||function(){(c[a].q=c[a].q||[]).push(arguments)};
+              t=l.createElement(r);t.async=1;t.src="https://www.clarity.ms/tag/"+i;
+              y=l.getElementsByTagName(r)[0];y.parentNode.insertBefore(t,y);
+          })(window, document, "clarity", "script", "jtrpzd10w5");
+        </script>
+      <!-- End Clarity Analytics -->
+    <% } %>
```

Comment:
```suggestion
    
```
Unless someone specifically requested to include this, I would prefer to leave it out. We don't need it for google analytics and a contract designer on tupaia web frontend asked us to setup it up a year ago but no one else has ever used it as far as I know

Reply:
There was a note on the ticket to maybe include it but we haven't had that confirmed yet, so I will remove for now